src/app/static/src/app/components/forms/DynamicFormMultiSelect.vue

```diff
@@ -35,7 +35,13 @@
         computed: {
             value: {
                 get() {
-                    return this.formControl.value ? this.formControl.value as string[] : []
+                    if (Array.isArray(this.formControl.value)) {
```

Comment:
Support default values provided either as a single string, or an array of strings.